client/data/authorizations/actions.ts

```diff
@@ -55,6 +72,77 @@ export function updateAuthorizationsSummary(
 	};
 }
 
+export function* submitCaptureAuthorization(
+	paymentIntentId: string,
+	orderId: number
+): Generator< unknown | Authorization > {
+	try {
+		yield dispatch( STORE_NAME, 'startResolution', 'getAuthorization', [
+			paymentIntentId,
+		] );
+
+		const result = yield apiFetch( {
+			path: `/wc/v3/payments/orders/${ orderId }/capture_authorization`,
+			method: 'post',
+			data: {
+				payment_intent_id: paymentIntentId,
+			},
+		} );
+
+		const authorization = {
+			payment_intent_id: ( result as CaptureAuthorizationApiResponse ).id,
+			captured:
```

Comment:
Shouldn't we also check an "outcome" here, since `status` will be `succeeded` for outcome `authorized` too.

Reply:
Thanks for checking it @dmvrtx. I looked into the [Stripe docs](https://stripe.com/docs/api/payment_intents/capture):

> Returns a PaymentIntent object with status="succeeded" if the PaymentIntent was capturable. Returns an error if the PaymentIntent was not capturable or an invalid amount to capture was provided.

According to that, it seems to me that we can rely that the payment was captured if the status is `succeeded` and there will be an error otherwise. What do you think?

Reply:
@dmvrtx I am merging the PR. Please let me know if you still have concerns and we can address it in a separate PR.

Reply:
For posterity, since I forgot to comment before clicking "Resolve": I agree with your choice here. I didn't connect it with the capture endpoint.